environment.yaml

```diff
@@ -4,6 +4,7 @@ channels:
 dependencies:
 - autopep8=1.5.2
 - beautifulsoup4=4.9.0
+- codecov=2.1.0
```

Comment:
See instructions: https://github.com/mozilla/OpenWPM/#managing-requirements
This file should not be manually edited.